releasenotes/notes/50328.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: release-notes/v2
+kind: feature
```

```suggestion
kind: bugfix
```

IMO this is just something that didn't work correctly before, and should have, and thus this is a bugfix - do you have any objection to classifying it as such?

---

No objections, changed.